Tests/Unit/ConfigModelTests.swift

```diff
@@ -0,0 +1,62 @@
+import Quick
+import Nimble
+@testable import RRemoteConfig
+
+class ConfigModelSpec: QuickSpec {
+    override func spec() {
+        context("initialization") {
+            describe("when json is the expected format") {
+                let data = """
+                    {"body":{"key1":"val1","key2":"val2","key3":"val3"},"keyId":"a1b2c3"}
+                    """.data(using: .utf8) ?? Data()
```

Comment:
Since Swift 5, you can use # for string containing " character:
let data = #"{"body":{"key1":"val1","key2":"val2","key3":"val3"},"keyId":"a1b2c3"}"#.data(using: .utf8) ?? Data()

Reply:
NIce!